[0m18:00:26.256180 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fe35347d590>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fe353b0cb10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fe35347d6d0>]}


============================== 18:00:26.259375 | 526c37df-03e2-4aec-8759-32eed308fe06 ==============================
[0m18:00:26.259375 [info ] [MainThread]: Running with dbt=1.7.1
[0m18:00:26.260073 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/turner/.dbt', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'fail_fast': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt init jaffle_shop', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m18:00:26.260659 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse dict {'dir': PosixPath('/home/turner/.dbt')}
[0m18:00:26.261149 [info ] [MainThread]: Creating dbt configuration folder at 
[0m18:00:26.262175 [debug] [MainThread]: Starter project path: /home/turner/.virtualenvs/dbt-tutorial/lib/python3.11/site-packages/dbt/include/starter_project
[0m18:00:26.264655 [info ] [MainThread]: 
Your new dbt project "jaffle_shop" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m18:00:26.265107 [info ] [MainThread]: Setting up your profile.
[0m18:01:17.507744 [error] [MainThread]: Encountered an error:

[0m18:01:17.514660 [error] [MainThread]: Traceback (most recent call last):
  File "/home/turner/.virtualenvs/dbt-tutorial/lib/python3.11/site-packages/dbt/cli/requires.py", line 90, in wrapper
    result, success = func(*args, **kwargs)
                      ^^^^^^^^^^^^^^^^^^^^^
  File "/home/turner/.virtualenvs/dbt-tutorial/lib/python3.11/site-packages/dbt/cli/requires.py", line 75, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/turner/.virtualenvs/dbt-tutorial/lib/python3.11/site-packages/dbt/cli/main.py", line 515, in init
    results = task.run()
              ^^^^^^^^^^
  File "/home/turner/.virtualenvs/dbt-tutorial/lib/python3.11/site-packages/dbt/task/init.py", line 344, in run
    self.setup_profile(profile_name)
  File "/home/turner/.virtualenvs/dbt-tutorial/lib/python3.11/site-packages/dbt/task/init.py", line 264, in setup_profile
    self.create_profile_from_target(adapter, profile_name=profile_name)
  File "/home/turner/.virtualenvs/dbt-tutorial/lib/python3.11/site-packages/dbt/task/init.py", line 181, in create_profile_from_target
    self.create_profile_from_profile_template(profile_template, profile_name)
  File "/home/turner/.virtualenvs/dbt-tutorial/lib/python3.11/site-packages/dbt/task/init.py", line 165, in create_profile_from_profile_template
    target = self.generate_target_from_input(prompts, initial_target)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/turner/.virtualenvs/dbt-tutorial/lib/python3.11/site-packages/dbt/task/init.py", line 131, in generate_target_from_input
    target[key] = click.prompt(
                  ^^^^^^^^^^^^^
  File "/home/turner/.virtualenvs/dbt-tutorial/lib/python3.11/site-packages/click/termui.py", line 164, in prompt
    value = prompt_func(prompt)
            ^^^^^^^^^^^^^^^^^^^
  File "/home/turner/.virtualenvs/dbt-tutorial/lib/python3.11/site-packages/click/termui.py", line 147, in prompt_func
    raise Abort() from None
click.exceptions.Abort

[0m18:01:17.519875 [debug] [MainThread]: Resource report: {"command_name": "init", "command_wall_clock_time": 51.265007, "process_user_time": 2.058988, "process_kernel_time": 0.12418, "process_mem_max_rss": "101460", "process_in_blocks": "400", "process_out_blocks": "80", "command_success": false}
[0m18:01:17.523954 [debug] [MainThread]: Command `dbt init` failed at 18:01:17.522763 after 51.27 seconds
[0m18:01:17.526200 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fe35347db90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fe357a1e610>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fe354a1b690>]}
[0m18:01:17.527622 [debug] [MainThread]: Flushing usage events
[0m18:01:35.678940 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fdd1cbf54d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fdd1cbf5810>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fdd1cbf5910>]}


============================== 18:01:35.682118 | 36113c9d-fedb-4a6a-ae6f-154a452e450a ==============================
[0m18:01:35.682118 [info ] [MainThread]: Running with dbt=1.7.1
[0m18:01:35.682796 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/turner/.dbt', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'warn_error': 'None', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'target_path': 'None', 'invocation_command': 'dbt init jaffle_shop', 'send_anonymous_usage_stats': 'True'}
[0m18:01:35.684111 [info ] [MainThread]: A project called jaffle_shop already exists here.
[0m18:01:35.685705 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 0.008746171, "process_user_time": 1.971536, "process_kernel_time": 0.099976, "process_mem_max_rss": "97636", "process_out_blocks": "8", "process_in_blocks": "0"}
[0m18:01:35.686533 [debug] [MainThread]: Command `dbt init` succeeded at 18:01:35.686335 after 0.01 seconds
[0m18:01:35.687063 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fdd1e19edd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fdd21122650>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fdd2105ea10>]}
[0m18:01:35.687639 [debug] [MainThread]: Flushing usage events
